blocks/api/paste.js

```diff
@@ -76,7 +78,9 @@ export function normaliseToBlockLevelNodes( nodes ) {
 }
 
 export default function( nodes ) {
-	return normaliseToBlockLevelNodes( nodes ).map( ( node ) => {
+	const prepare = compose( [ normaliseToBlockLevelNodes, gDocs, stripAttributes ] );
```

Comment:
From what I've seen TinyMCE offers [`PastePreProcess`](https://www.tinymce.com/docs/advanced/events/#pastepreprocess) and [`PastePostProcess`](https://www.tinymce.com/docs/advanced/events/#pastepostprocess) events (we are currently using the second one). The `PastePreProcess` event is just the same paste data but as an HTML string, so it didn't offer any extra info in my tests. 

If we wind up needing some stripped metadata I can re-add a separate test, but I suspect we may not need it.